apps/designer/app/shared/props-utils/update-props.ts

```diff
@@ -4,12 +4,13 @@ import {
 } from "@webstudio-is/react-sdk";
 
 export const updateAllUserPropsMutable = (
+  treeId: string,
   allUserProps: AllUserProps,
-  { instanceId, propsId, treeId, updates }: UserPropsUpdates
+  { instanceId, propsId, updates }: UserPropsUpdates
 ) => {
   if (instanceId in allUserProps === false) {
     allUserProps[instanceId] = {
-      id: propsId,
```

Comment:
In my uderstanding if we start to generate Id here, there must be some place there we generated it before, but I dont see the removed line with previous generated Id

Reply:
Here It's still generated. I just removed this method call in one place https://github.com/webstudio-is/webstudio-designer/blob/main/packages/project/src/shared/props/create-instance-props.ts#L13